index.html

```diff
@@ -45,5 +45,181 @@ <h2>I'm a Software Developer</h2>
+            <div class="details">
+                <h2>Profesional Art Printing Data</h2>
+                <p class="data-arc">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                </p>
+                <div class="programs">
+                        <ul class="position-items">
+                            <li class="stack-position">html </li>
+                            <li class="stack-position1">bootstrap </li>
+                            <li class="stack-position">Ruby</li>
+                        </ul>
+                </div>
+                <button class="see-project-btn">See Project </button>
+            </div>
+            <!-- -----------3-------------- -->
+            <div class="details">
+                <h2>Profesional Art Printing Data</h2>
+                <p class="data-arc">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                </p>
+                <div class="programs">
+                    <ul class="position-items">
+                        <li class="stack-position">html </li>
+                        <li class="stack-position1">bootstrap </li>
+                        <li class="stack-position">Ruby</li>
+                    </ul>  
+                </div>
+                <button class="see-project-btn">See Project </button>
+            </div>
+
+            <!-- ------4----- -->
+                <div class="details">
+                    <h2>Profesional Art Printing Data</h2>
+                    <p class="data-arc">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <div class="programs">
+                        <ul class="position-items">
+                            <li class="stack-position">html </li>
+                            <li class="stack-position1">bootstrap </li>
+                            <li class="stack-position">Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="see-project-btn">See Project </button>
+                </div>
+                <!-- --------5-------- -->
+                <div class="details">
+                    <h2>Profesional Art Printing Data</h2>
+                    <p class="data-arc">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <div class="programs">
+                        <ul class="position-items">
+                            <li class="stack-position">html </li>
+                            <li class="stack-position1">bootstrap </li>
+                            <li class="stack-position">Ruby</li>
+                        </ul>
```

Comment:
Good job implementing the `ul` and `li` as suggested by the previous reviewer. 👍